builtin/mainmenu/content/contentdb.lua

```diff
@@ -129,6 +145,15 @@ local function start_install(package, reason)
 					end
 					conf:set("author",     package.author)
 					conf:set("release",    package.release)
+					if package.aliases then
+						local gameid_alias = ""
```

Comment:
I'd prefer putting these in a table and doing `table.concat(gameid_aliases, ",")` to concat them in the end. (I find that more readable. It also doesn't produce a trailing comma.)